Assignment-2-User-Management.c

```diff
@@ -0,0 +1,175 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+//defining the struct to store the details of the user
+struct user
+{
+    int id;
+    char name[50];
+    int age;
+};
+
+void main()
+{
+
+    int ch;
+
+    while (1)
+    {
+        printf(" \n__________________User Management System______________________     \n\n");
+
+        printf(" 1. Add new User \n 2. Display all users\n");
+        printf(" 3. Update user  \n 4. Delete User\n 5. Exit\n");
+
+        printf("\nEnter your Choice : ");
+        scanf("%d", &ch);
+
+        switch (ch)
+        {
+        case 1:  // to create new user
+        {
+            struct user newuser;
+
+            printf("\nUser ID :");
+            scanf("%d", &newuser.id);
+
+            printf("User Name:");
+            scanf(" %[^\n]", newuser.name);
+
+            printf("User Age : ");

```

Comment:
can we handle read input and printing output in separate function? take care at all places.